code.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "project-ticket-template",
+  "description": "Creates ticket templates within GitHub project boards.",
+  "longDescription": "A longer description of the project.",
```

Comment:
Update value for `longDescription`. Make sure it is a string of 150 to 10000 characters